code/__HELPERS/unsorted.dm

```diff
@@ -1520,7 +1520,7 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			. += V
 
 //datum may be null, but it does need to be a typed var
-#define NAMEOF(datum, X) (list(##datum.##X, #X)[2])
+#define NAMEOF(datum, X) (FALSE ? ##datum.##X : #X)
```

Comment:
what exactly is the point of this ternary

Reply:
It has to have the ##datum.##X in there somewhere to guarantee it actually does exist